charts/self-host/templates/ingress.yaml

```diff
@@ -18,6 +18,8 @@ metadata:
 {{- with .Values.general.ingress.labels }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+  annotations:
+{{- include "bitwarden.annotations" . | indent 4 }}
```

Comment:
It looks like this may have been missed.  The `ingress.yaml` file has two `annotations` keys.  `Line 21` should be deleted and `Line 22` should be moved above `line 12`.